h/streamer/filter.py

```diff
@@ -11,29 +10,28 @@
 SCHEMA = {
     "type": "object",
     "properties": {
-        "name": {"type": "string", "optional": True},
+        # Ignored, but kept here for backwards compatibility.
         "match_policy": {
             "type": "string",
-            "enum": ["include_any", "include_all",
-                     "exclude_any", "exclude_all"]
+            "enum": ["include_any"]
         },
+
+        # Ignored, but kept here for backwards compatibility.
```

Comment:
Given that both `match_policy` and `actions` are technically ignored, should they maybe be removed from the `required` list here https://github.com/hypothesis/h/pull/5413/files#diff-29ecaf3429ec998affde6811c0455e51R38 ?

Reply:
Yes, although we might want to go straight to re-doing the remaining API entirely to the format described here which Nick proposed for a public real-time API: https://h.readthedocs.io/en/latest/api/realtime/#sending-messages